### PR TITLE
Repositories

### PR DESCRIPTION
**Pull request by anancarv**

### CODE REVIEW

File: pyartifactory/objects.py [hunk 1]
```diff
@@ -262,7 +276,12 @@ def list(self) -> GroupList:
             request_url, auth=self._auth, verify=self._verify, cert=self._cert
         )
         r.raise_for_status()
-        return GroupList(groups=r.json())
+
+        group_list = []
+        for group in r.json():
```
Comment (nymous) [inline pyartifactory/objects.py]: List comprehension?

File: pyartifactory/objects.py [hunk 2]
```diff
@@ -460,19 +536,27 @@ def list(self) -> RepositoryList:
             request_url, auth=self._auth, verify=self._verify, cert=self._cert
         )
         r.raise_for_status()
-        return RepositoryList(repositories=r.json())
+
+        repositories_list = []
+        for repository in r.json():
```
Comment (nymous) [inline pyartifactory/objects.py]: List comprehension?

File: tests/test_security.py [hunk 3]
```diff
@@ -0,0 +1,80 @@
+import responses
```
Comment (nymous) [inline tests/test_security.py]: You don't assert anything in this test file, is this OK?

File: README.md [hunk 4]
```diff
@@ -97,7 +109,42 @@ art.security.get_api_key(             art.security.regenerate_api_key(      art.
...
+```
+
+Get a single repository (Local, Virtual or Remote):
+```python
+local_repo = art.groups.get_local_repo("local_repo_name")
+virtual_repo = art.groups.get_virtual_repo("virtual_repo_name")
+remote_repo = art.groups.get_remote_repo("remote_repo_name")
+```
+
+Create/Update a group:
```
[the remark this answers is not in the document]
Comment (anancarv) [reply README.md]: Corrected in another pull request

File: pyartifactory/models/Repository.py [hunk 5]
```diff
@@ -54,6 +54,30 @@ class PomRepoRefCleanupPolicy(str, Enum):
...
+    bitbucket = "BITBUCKET"
+    oldstash = "OLDSTASH"
+    stash = "STASH"
+    artifactory = "ARTIFACTORY"
+    custom = ("CUSTOM",)
+
+
+class ContentSynchronisation(BaseModel):
+    enabled: bool = False
+    statistics: Dict[str, bool] = {"enabled": False}
```
[the remark this answers is not in the document]
Comment (anancarv) [reply pyartifactory/models/Repository.py]: Corrected in another pull request

File: pyartifactory/models/Repository.py [hunk 6]
```diff
@@ -54,6 +54,30 @@ class PomRepoRefCleanupPolicy(str, Enum):
...
+    properties: Dict[str, bool] = {"enabled": False}
+    source: Dict[str, bool] = {"originAbsenceDetection": False}
+
+
+class Nuget(BaseModel):
+    nuget: Dict[str, str] = {
```
[the remark this answers is not in the document]
Comment (anancarv) [reply pyartifactory/models/Repository.py]: Corrected in another pull request

File: pyartifactory/objects.py [hunk 7]
```diff
@@ -90,9 +95,12 @@ def list(self) -> UserList:
             request_url, auth=self._auth, verify=self._verify, cert=self._cert
         )
         r.raise_for_status()
-        return UserList(users=r.json())
+        user_list = []
+        for user in r.json():
```
[the remark this answers is not in the document]
Comment (anancarv) [reply pyartifactory/objects.py]: Corrected in another pull request

File: pyartifactory/objects.py (hunk 1 above)
Comment (anancarv) [reply pyartifactory/objects.py]: Corrected in another pull request

File: pyartifactory/objects.py (hunk 2 above)
Comment (anancarv) [reply pyartifactory/objects.py]: Corrected in another pull request

File: tests/test_security.py (hunk 3 above)
Comment (anancarv) [reply tests/test_security.py]: Yes it is. The only things I can assert are:
  - The HTTP Get request have been made successfully  
  - The returned value of the request
However, those are asserted by the responses package

**Pull request merged**